### PR TITLE
Diego flores

### PR DESCRIPTION
**Pull request by Dfloresa**
Added day 2 challenge

### CODE REVIEW

File: DiegoFlores/Day2/Challenge/Main.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package person;
+
+public class Main {
+
+	public static void main(String[] args) {
+		
+		Student student1 = new Student(000001,"architecture",1);
```
Comment (luisalanmz) [inline DiegoFlores/Day2/Challenge/Main.java:7]: This string and all those that are in this class can go in constants

**Review by luisalanmz — CHANGES_REQUESTED** [review]

File: DiegoFlores/Day2/HackerRank/Inheritance1/Main.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package animal;
+
+public class Main {
+
+	public static void main(String[] args) {
```
Comment (luisod78) [inline DiegoFlores/Day2/HackerRank/Inheritance1/Main.java:5]: Please fix identation

**Review by luisod78 — CHANGES_REQUESTED** [review]

**Review by Dfloresa — COMMENTED** [review]
Fixed indentation and constant strings

**Review by Dfloresa — COMMENTED** [review]
Fixes done

File: DiegoFlores/Day2/Challenge/Main.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
+package person;
+
+    	public static final String STUDENT1CAREER = "architecture";
```
Comment (globant-bootcamp) [inline DiegoFlores/Day2/Challenge/Main.java:3]: For the final name variables the words will need to by separeted by "_" example: STUDENT_ONE_CARRER, fix all the names

File: DiegoFlores/Day2/Challenge/Main.java [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+	public static final String TEACHER1NAME = "Obi-Wan";
+	public static final String TEACHER1SURNAME = "Kenobi";
+
+
+public class Main {
+
+	public static void main(String[] args) {
+		
+		Student student1 = new Student(000001,"architecture",1);
+			student1.setNames("John");
```
Comment (globant-bootcamp) [inline DiegoFlores/Day2/Challenge/Main.java:22]: What is the reason to not use the final variables that you alredy have? change all

File: DiegoFlores/Day2/Challenge/Person.java [hunk 5]
```diff
@@ -0,0 +1,31 @@
+package person;
+
+public class Person {
+
+	private String names;
+	private String surnames;
+	private int age;
+	
+
```
Comment (globant-bootcamp) [inline DiegoFlores/Day2/Challenge/Person.java:9]: Remove whitespace

File: DiegoFlores/Day2/Challenge/Person.java [hunk 6]
```diff
@@ -0,0 +1,31 @@
+package person;
+
+public class Person {
+
+	private String names;
+	private String surnames;
+	private int age;
+	
+	
+
```
Comment (globant-bootcamp) [inline DiegoFlores/Day2/Challenge/Person.java:10]: Same

File: DiegoFlores/Day2/Challenge/Person.java [hunk 7]
```diff
@@ -0,0 +1,31 @@
+package person;
+
+public class Person {
+
+	private String names;
+	private String surnames;
```
Comment (globant-bootcamp) [inline DiegoFlores/Day2/Challenge/Person.java:6]: surnames? what is sur? improve the class name and use lowerCamelCase example userName

File: DiegoFlores/Day2/Challenge/Student.java [hunk 8]
```diff
@@ -0,0 +1,41 @@
+package person;
+
+public class Student extends Person {
+
+	private int studentId;
+	private String	career;
```
Comment (globant-bootcamp) [inline DiegoFlores/Day2/Challenge/Student.java:6]: Remove the extra space between String and carrer

File: DiegoFlores/Day2/Challenge/Student.java [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+	private int semester;
+	
+
```
Comment (globant-bootcamp) [inline DiegoFlores/Day2/Challenge/Student.java:9]: Remove whitespace

File: DiegoFlores/Day2/Challenge/Student.java [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+	}
+	
+	public Student(int studentId, String career, int semester) {
+		super();
+		this.studentId = studentId;
+		this.career = career;
+		this.semester = semester;
+	}
+	
+
```
Comment (globant-bootcamp) [inline DiegoFlores/Day2/Challenge/Student.java:21]: Remove whitespace

File: DiegoFlores/Day2/Challenge/Teacher.java [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+	public void setCourse(String course) {
+		this.course = course;
+	}
+	public String getAgreementTime() {
+		return agreementTime;
+	}
+	public void setAgreementTime(String agreementTime) {
+		this.agreementTime = agreementTime;
+	}
+
```
Comment (globant-bootcamp) [inline DiegoFlores/Day2/Challenge/Teacher.java:31]: Remove whitespace

File: DiegoFlores/Day2/Challenge/Teacher.java [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+		this.course = course;
+	}
+	public String getAgreementTime() {
+		return agreementTime;
+	}
+	public void setAgreementTime(String agreementTime) {
+		this.agreementTime = agreementTime;
+	}
+	
+
```
Comment (globant-bootcamp) [inline DiegoFlores/Day2/Challenge/Teacher.java:32]: Remove whitespace

File: DiegoFlores/Day2/HackerRank/Inheritance1/Animal.java [hunk 13]
```diff
@@ -0,0 +1,7 @@
+package animal;
+
+public class Animal {
+	void walk(){
+		System.out.println("I am walking");
```
Comment (globant-bootcamp) [inline DiegoFlores/Day2/HackerRank/Inheritance1/Animal.java:5]: Put this hardcode text inside a final static variable

File: DiegoFlores/Day2/HackerRank/Inheritance1/Bird.java [hunk 14]
```diff
@@ -0,0 +1,13 @@
+package animal;
+
+public class Bird extends Animal{
+	
+	void fly(){
+		System.out.println("I am flying");
```
Comment (globant-bootcamp) [inline DiegoFlores/Day2/HackerRank/Inheritance1/Bird.java:6]: Put this hardcode text inside a final static variable

File: DiegoFlores/Day2/HackerRank/Inheritance1/Bird.java [hunk 15]
```diff
@@ -0,0 +1,13 @@
...
+	}
+    
+    void sing(){
+        System.out.println("I am singing");
```
Comment (globant-bootcamp) [inline DiegoFlores/Day2/HackerRank/Inheritance1/Bird.java:10]: Put this hardcode text inside a final static variable

File: DiegoFlores/Day2/HackerRank/Inheritance1/Main.java [hunk 16]
```diff
@@ -0,0 +1,13 @@
...
+		
+		 Bird bird = new Bird();
```
Comment (globant-bootcamp) [inline DiegoFlores/Day2/HackerRank/Inheritance1/Main.java:7]: Please fix identation

File: DiegoFlores/Day2/HackerRank/Inheritance2/Solution.java [hunk 17]
```diff
@@ -0,0 +1,29 @@
...
+
+
+class Adder extends Arithmetic{
+    
+}
+
+public class Solution{
+    public static void main(String []args){
+        // Create a new Adder object
+        Adder a = new Adder();
```
Comment (globant-bootcamp) [inline DiegoFlores/Day2/HackerRank/Inheritance2/Solution.java:21]: Improve name variable

**Review by globant-bootcamp — CHANGES_REQUESTED** [review]

**Review by Dfloresa — COMMENTED** [review]
Fixes done

**Review by Dfloresa — COMMENTED** [review]
More fixes done in classes (indentation)

**Review by globant-bootcamp — APPROVED** [review]

**Pull request merged**